content/html/concepts/attributes/terms/id/id.md

```diff
@@ -21,6 +21,28 @@ The HTML global **`id`** attribute is used to specify a unique identifier for an
 
 The syntax consists of `id` followed by the assignment operator (`=`) and an `id` name in quotation marks (`" "`). The `id` name is case-sensitive and must contain at least one character. It cannot start with a number or contain whitespaces.
 
+## Unique ID Attributes
+
+Specific and unique `id` attributes can be assigned to elements to differentiate them, and `id` values can be referenced in CSS and JavaScript to manipulate, format, and perform specific instructions on a single element alone, without affecting others of the same type.
+
+Valid `id` attribute values begin with a letter and must comprise only letters (`a`-`Z`), digits (`0`-`9`), hyphens (`-`), underscores (`_`), and periods (`.`).
+
+```html
+<h1 id="th1s_is.v4lid">Hello World</h1>
+```
+
+**Note:** While `th1s_is.v4lid` is indeed a valid `id` value, it's best to use a consistent naming scheme such as camel case (`thisIsValid`), dash case (`this-is-valid`), or snake case (`this_is_valid`).
```

Comment:
```suggestion
> **Note:** While `th1s_is.v4lid` is indeed a valid `id` value, it's best to use a consistent naming scheme such as camel case (`thisIsValid`), dash case (`this-is-valid`), or snake case (`this_is_valid`).
```
This is bringing the old entry in line with the current style for "notes"

Reply:
added > sign to fix the formatting.